docs/apache-airflow/index.rst

```diff
@@ -110,7 +110,7 @@ unit of work and continuity.
 
     Operators and hooks <operators-and-hooks-ref>
     CLI <cli-and-env-variables-ref>
-    Macros <macros-ref>
+    Templates <templates-ref>
```

Comment:
I decided to rename the page, as it was already talking about more than just the macros, but by adding filters there too it is even more true.

Reply:
Why not name it something like "context" or "task context"?

Reply:
Because macros and filters are not anything to do with task context  - only the variables are.